docs/images.md

```diff
@@ -115,7 +118,7 @@ metadata:
 spec:
   containers:
     - name: foo
-      image: registry.example.com/bar/fo
+      image: janedoe/awesomeapp:v1
```

Comment:
I've tested these examples.  It took a good chunk of a day to get it all working.  I'd like to check in this improvement.